commons/src/main/java/io/hyscale/commons/config/SetupConfig.java

```diff
@@ -63,9 +63,7 @@ private static String getAbsolutePath() {
     }
 
     public static void clearAbsolutePath() {
-        if (absolutePathTL != null) {
-            absolutePathTL.remove();
-        }
+        absolutePathTL.remove();
```

Comment:
Why is the null check removed?

Reply:
absolutePathTL will not be null since we are initializing it, the value it contains might be, hence removed the null check. This was also suggested by sonarcloud.